jihyunkim-api/models/signupDao.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-unused-vars */
+import prisma from '../prisma';
+import bcrypt from 'bcrypt';
+
+const userSignUp = async (req) => {
+  const { email, name, password } = req.body;
+
+  if (!email || !password) {
+    const error = new Error('필수 정보를 입력해주세요 ☄️ ');
+    error.statusCode = 400;
+    throw error;
+  }
+
+  const saltRounds = 10;
+
+  const salt = bcrypt.genSaltSync(saltRounds);
+  const hashedPassword = bcrypt.hashSync(password, salt);
+
+  const createdUser = await prisma.$queryRaw(`
+        INSERT INTO users(email, name, password) 
+        VALUES ('${email}', '${name}','${password}')
+      `);
+  return createdUser;
+};
+
+const userLogin = async (req, res) => {
+  const { email, password } = req.body;
+  const usersRegister = await prisma.users.findUnique({ where: { email } });
```

Comment:
```suggestion
  const userRegistered = await prisma.users.findUnique({ where: { email } });
```
문맥상 이 변수명이 더 어울릴 것 같습니다

Reply:
네 수정 완료했습니다!